lib/lbt/resources/LibraryFileAnalyzer.js

```diff
@@ -49,7 +48,7 @@ function createRawInfo(rawModule) {
 	}
 }
 
-function getDependencyInfos( name, content ) {
+export default function getDependencyInfos( name, content ) {
```

Comment:
I feel like the module name doesn't match with the functionality.
I'd rather expect to have a named export here.